--- a/Makefile-options	2021-04-11 17:01:59.000000000 +0200
+++ b/Makefile-options	2021-12-06 19:07:21.034134116 +0100
@@ -271,13 +271,7 @@
 	endif
 endif
 
-ifeq ($(findstring bundledlibcpuid,$(config)),bundledlibcpuid)
-	USE_BUNDLED_LIBCPUID = True
-else ifeq ($(findstring systemlibcpuid,$(config)),systemlibcpuid)
-	USE_BUNDLED_LIBCPUID = False
-else
-	USE_BUNDLED_LIBCPUID = True
-endif
+USE_BUNDLED_LIBCPUID = False
 
 ifeq ($(findstring bundledlibcurl,$(config)),bundledlibcurl)
 	USE_BUNDLED_LIBCURL = True
@@ -337,13 +331,7 @@
 	endif
 endif
 
-ifeq ($(findstring bundledlibwebp,$(config)),bundledlibwebp)
-	USE_BUNDLED_LIBWEBP = True
-else ifeq ($(findstring systemlibwebp,$(config)),systemlibwebp)
-	USE_BUNDLED_LIBWEBP = False
-else
-	USE_BUNDLED_LIBWEBP = True
-endif
+USE_BUNDLED_LIBWEBP = False
 
 ifeq ($(findstring bundledlibxml2,$(config)),bundledlibxml2)
 	USE_BUNDLED_LIBXML2 = True
